integration/kubernetes/helm-chart/alluxio/templates/csi/client-sc-nginx.yaml

```diff
@@ -0,0 +1,61 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+{{ if .Values.csi.clientEnabled -}}
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
```

Comment:
It would be better to abstract the storageClass to an individual file

Reply:
Fixed